### ISSUE TITLE
Add instructions to documentation on how to run tests with pytest-bdd

### ISSUE THREAD

**Issue by k2m30** [pytest-dev/pytest-bdd#639]
I am not the only one struggling with it. Here is Stackoverflow [discussion](https://stackoverflow.com/questions/54751475/how-to-run-a-pytest-bdd-test) started almost five years ago and still haven't solved.

**Comment by jzohrab** [comment]
I agree that a simple note in the readthedocs would help.  The readme in this repo has an example.

Define your step definitions in a test_xxxx.py file, and your feature file somewhere, with a reference to that file in your step defs.  Then when your run pytest, it will pick up the test and run it just like the others.
